Resources/Prototypes/_RMC14/Roles/Jobs/Other/synth.yml

```
@@ -0,0 +1,87 @@
+- type: job
+  parent: CMJobBase
+  id: RMCJobSynthetic
+  name: rmc-job-name-synth
+  description: rmc-job-description-synth
+  playTimeTracker: RMCJobSynthetic
+  ranks:
+    RMCRankSyntheticMG:
+    - !type:RoleTimeRequirement
+      role: RMCJobSynthetic
+      time: 252000 # 70 hours
+    RMCRankSynthetic: []
```

Checking in-game it's master sergeant despite the code saying otherwise, not sure on the specifics, so leaving it on this for now until we figure it out
```suggestion
    RMCRankMasterSergeant: [ ]
```